packages/fxa-admin-server/nest-cli.json

```diff
@@ -2,8 +2,18 @@
   "collection": "@nestjs/schematics",
   "sourceRoot": "src",
   "compilerOptions": {
-    "assets": ["**/*.gql"],
-    "watchAssets": true
+    "assets": [
+      {
+        "include": "**.*.gql",
```

Comment:
This looks like an interesting glob.
Should this be `**.*.gql` or `**/*.gql`?

Reply:
I think `**.*.gql` matches files like `test.query.gql` but the other does not?

Reply:
Was a typo. Fixed.